tests/unit/test_cubeset.py

```diff
@@ -0,0 +1,176 @@
+# encoding: utf-8
+
+"""Unit test suite for `cr.cube.cube.Cubeset` class."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+import numpy as np
+
+from cr.cube.cube import CubeSet, Cube
+from cr.cube.cubepart import _Slice, _Strand, _Nub
+from cr.cube.enum import DIMENSION_TYPE as DT
+
+from ..unitutil import instance_mock, property_mock
+
+
+class DescribeCrunchCubeSet(object):
+    def it_knows_whether_it_has_means(self, has_means_fixture, _cubes_prop_, cube_):
+        first_cube_has_means, expected_value = has_means_fixture
+        cube_.has_means = first_cube_has_means
+        _cubes_prop_.return_value = (cube_,)
+        cube_set = CubeSet(None, None, None, None)
+
+        has_means = cube_set.has_means
+
+        assert has_means == expected_value
+
+    def it_knows_whether_it_has_weighted_counts(
+        self, has_weighted_counts_fixture, _cubes_prop_, cube_
+    ):
+        first_cube_has_w_counts, expected_value = has_weighted_counts_fixture
+        cube_.is_weighted = first_cube_has_w_counts
+        _cubes_prop_.return_value = (cube_,)
+        cube_set = CubeSet(None, None, None, None)
+
+        is_weighted = cube_set.has_weighted_counts
+
+        assert is_weighted == expected_value
+
+    def it_knows_its_name(self, has_name_fixture, _cubes_prop_, cube_):
```

Comment:
I don't think many of these really benefit from a fixture because there is no real logic in the method. Returning "foobar" or `None` is not really different in this case because it just passes back what the name of the first cube is. So in these cases we can dispense with the fixture and just pick a representative case like "Beverage" and just test the one case.

Fixtures are great when you need to test multiple cases, but they do obscure the test somewhat because they are an "effect at a distance" and to really understand the test you need to see the fixture. So we should only use them when we need them, which is generally to exercise some branch (or implicit branch) in the code.